crates/model/src/order.rs

```diff
@@ -585,8 +596,22 @@ pub struct OrderMetadata {
     #[serde(flatten)]
     pub class: OrderClass,
     pub settlement_contract: H160,
+    /// This is 0 for liquidity orders as they should only ever be used to
```

Comment:
```suggestion
    /// This is `fee_amount` for liquidity orders as they should only ever be used to
```

Reply:
How we populate `full_fee_amount` didn't change. We still default initialize it for liquidity orders. But I guess in light of the discussion about bribing the protocol this value should never be less than `fee_amount`.

Reply:
Since this is a slight change coming from your comment could you please check if you agree with this change: [c4e6970](https://github.com/cowprotocol/services/pull/1199/commits/c4e6970eef0c73e2cd10ba82a5236b8a2ce6144d)

Reply:
Looks good.